tests/src/common/lpp.rs

```diff
@@ -1,3 +1,4 @@
+use currencies::Lpns;
```

Comment:
[concerns all the changes in the integration tests] Similarly to the contracts, please check if we have common definitions of the groups. Consider the top-most code to refer to a common definition and pass it down to the lower-level code.